### DISCUSSION TITLE
print python output as command tool output

### DISCUSSION THREAD

**Discussion by astrung** [openai/whisper#1542] [Q&A]
Hello. I am getting used to this library, and I know this library can be used with command line or python API. However, now my code with python API can only print text, but I want to get timestamp of this text too. In other words, I want to get same output as console output, not just print text.
* My current python output: `Nowadays, we are so used to getting food whenever....`
* My expected output(command line output): `[00:14.960 --> 00:20.320]  Nowadays, we are so used to getting food whenever...`

SO how can I adjust my python code to do it ? Please help
My current code (tutorial code):

    import whisper
    model = whisper.load_model("medium")
    result = model.transcribe("eng_test_audio.mp3")
    print(result["text"])

**Comment by Purfview** [comment]
I made such output in [Whisper-Faster](https://github.com/Purfview/whisper-standalone-win), but that is for Windows and not Python.

**Reply by astrung** [reply]
@Purfview I think you are farmiliar with building command-line, so would you point out for me which main script is using for building whisper command line ? I think if i can look at script which is used to built command-line tool, it will be helpful too

**Comment by phineas-pta** [comment]
try manipulate `result["segments"]`

**Reply by astrung** [reply]
@phineas-pta Using `result["segments"]` is still too complex, its output is a collective of timestamps and tokens, and I need to adjust it by my hands. 

Because command-line output is already nice format, I think this library have support this kind of output too, but i don't know how to find it in document or code. May be looking at command-line main script will help. Do you which is their main script for running in command-line ?

**Reply by ryanheise** [reply]
Isn't the SRT output also in a nice format? It's not identical but it is very similar to the console output.
